pkgs/stdenv/darwin/default.nix

```diff
@@ -15,15 +16,14 @@
     cpio    = fetch { file = "cpio";  sha256 = "0ms5i9m1vdksj575sf1djwgm7zhnvfrrb44dxnfh9avr793rc2w4"; };
     tarball = fetch { file = "bootstrap-tools.cpio.bz2"; sha256 = "1lz1b0grl4642h6n635xvi6imf0yyy1zyzdr9ing5aphzz0z5iic"; executable = false; };
   }
+, ...
```

Comment:
I don't think so. the `...`s allows us to give the all stdenvs the same arguments not worrying about which subset they actually use. See the simplifications to `stdenv/default.nix` in the same commit that introduced this.